pkg/apis/paddlepaddle/v1/trainingjob.go

```diff
@@ -0,0 +1,32 @@
+package v1
```

Comment:
I noticed that in many of our Go repos, there is a directory `pkg`. I have never seen such a convention in other Go repos -- not in any well-known libraries like [Google's API's Go client](https://github.com/google/google-api-go-client/blob/master/sheets/v4/sheets-gen.go), not [Go lang's repo itself](https://github.com/golang/go/tree/master/src).

This convention would make it hard for other Go projects to import ours. The authors have to write:

```go
import "github.com/paddlepaddle/edl/pkg/apis/paddlepaddle/v1"
```

Please be aware that there are two `paddlepaddle` in the above import path. Ths is pretty redundant.

To make this import path concise and clear, we might want

```go
import "github.com/paddlepaddle/edl/v1"
```

To achive this, we should have `training_job.go` in the `/v1`, instead of `/pkg/apis/paddlepaddle/v1`.

Reply:
`pkg` directory is widely used in many go repos like https://github.com/coreos/etcd, https://github.com/kubernetes/kubernetes, and the intention is to hide internal implement. I think we can keep the `pkg` directory and put `cmd/edl` to `edl` is more general. Currently since edl does not have any API provided for users to call, we don't have to expose `apis` directory, `/pkg/apis/paddlepaddle/v1` is only for internal use and call the k8s API.